pkg/lib/k8s/quantity.go

```diff
@@ -133,6 +132,10 @@ type quantityMarshalable struct {
 	UserString string
 }
 
+func (quantity Quantity) MarshalYAML() (interface{}, error) {
```

Comment:
To me it seems risky that `MarshalYAML()` behaves differently from `MarshalJSON()`. Maybe they can both return `quantity.String()`? Let's discuss